### ISSUE TITLE
(WEB REPORT BY: palamag REMOTE: 198.245.63.50:7777) Disappearing iron in recycling furnace

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#5349]
>Revision 8068683f81778067e22ed536f91e4905a4f6b72d

> Upon sticking a stack of 50 units of iron sheets in front of the recycler, they disappeared and did not show up in the recycling menu. This prevents iron to be recycled and prevents plasteel from being created.

> Iron is correctly inputed in the recycler.

> The iron did not appear in the recycler.

> Issue might be related to previously recycled items being inputed. When this issue occured, there was 0.14667 units of iron in the recycler, probably from previously recycled misc items.

**Comment by PJB3005** [comment]
Blame @ComicIronic

**Comment by Shadowmech88** [comment]
>No it doesn't.

Personally, I've tried to make plasteel on both the test server and the live server and both times the metal disappeared and the plasma was spat back out. I saw that video you posted in IRC of you making plasteel, but in that same video it shows there being 50 metal inside the recycler, whereas with this bug, the metal never shows up in the recycler, it just disappears.
If this is the same bug I've been encountering, then it does prevent plasteel from being made, since plasteel production requires there being metal inside the furnace and this bug prevents metal from ever registering as being inside the furnace.
